frontend/src/components/ShootWorkers/ManageWorkers.vue

```diff
@@ -300,6 +307,9 @@ export default {
       this.newShootWorkerCIDR = newShootWorkerCIDR
       this.kubernetesVersion = kubernetesVersion
       this.initialZones = uniq(flatMap(workers, 'zones'))
+      this.$nextTick(() => {
```

Comment:
Why do we need this `$nextTick`? Add a comment that explains why it is needed.

Reply:
added comment